frontend/app/api/concepts.json

```diff
@@ -59,10 +59,20 @@
             {
               "id": "budget",
               "label": "Budget",
-              "type": "INTEGER_RANGE",
+              "type": "MONEY_RANGE",
               "unit": "1K USD",
```

Comment:
This doesn't make sense when we're now entering Euro values.

Reply:
This will render the correct filter type.

Reply:
I meant the `"unit": "1K USD"` line, I'll fix this online